### PR TITLE
[PRP-253] Fixes from live UAT on 2023-03-22

### PR DESCRIPTION
**Pull request by rocketnova**
## Ticket

https://wicmtdp.atlassian.net/browse/PRP-253

- Generally: 
  - Switch all cases of `t()` to `<Trans>` where possible to support basic html tags in i18n strings
  - Enable support of `<em>` tags in i18n strings
  - Swap `<div>` tags around paragraph content to `<p>` tags
  - Alphabetize sections in `en/common.json`
  - Remove hard line break in site name
  - Change font from Source Sans Pro to Public Sans
  - Update body line height to 1.5em
  - Create a `.intro` class for intro paragraphs on pages
  - Create a `.legend-large` and `.label-large` class for our large labels/legends
  - Adjust misc spacing
- Components:
  - AdjunctiveInput: Use new `<List>` component
  - DateInput: Support optionally setting the input as required fields
  - ParticipantCard: Support optionally setting the `<AdjunctiveInput>`, `<DateInput>`, and `<RelationshipInput>` components as required fields
  - RequiredQuestionStatement: Add new component for the help text that explains required fields have asterisks
  - TextField: Support optional `className` argument
- Pages:
  - Add `<RequiredQuestionStatement>` to all pages with required fields
  - Count: Use `type=number`, set max width on the input field, use the `.label-large` class
  - Details: Make all fields required except for preferred name, turn on date hint, turn on preferred name input, add missing i18n strings
  - Name: Remove extra content

These changes came out of our Live UAT session on 2023-03-22.

Review the chromatic storybook

### CODE REVIEW

File: participant/app/routes/$localAgency/recertify/count.tsx [hunk 1]
```diff
@@ -6,41 +6,46 @@ import { Trans, useTranslation } from "react-i18next";
...
+import { RequiredQuestionStatement } from "~/components/RequiredQuestionStatement";
 
 export default function Count() {
   const { t } = useTranslation();
   const householdSizeProps: TextFieldProps = {
     id: "householdSize",
     type: "input",
-    inputType: "text",
+    inputType: "number",
     labelKey: "Count.householdSize.label",
```
Comment (makaelastephens) [inline participant/app/routes/$localAgency/recertify/count.tsx:17]: This label should be "large"

File: participant/styles/_uswds-theme.scss [hunk 2]
```diff
@@ -16,5 +16,6 @@ in the form $setting: value,
   $theme-font-path: '/uswds/fonts',
   $theme-image-path: '/uswds/img',
   $theme-type-scale-lg: 7,
-  $theme-header-logo-text-width: 100%
+  $theme-header-logo-text-width: 100%,
```
Comment (makaelastephens) [inline participant/styles/_uswds-theme.scss:18]: I don't have this pulled up on my local environment so maybe this isn't accurate, but using the Chromatic Storybook "small mobile" view, the header doesn't look as expected. I'd expect the blue background to extend the entire width. 
image

**Review by makaelastephens — COMMENTED** [review]
Can we also update the line height, [per this convo](https://nava.slack.com/archives/C03CZ4JKNQY/p1679596198800349?thread_ts=1679520428.801179&cid=C03CZ4JKNQY)?

File: participant/app/routes/$localAgency/recertify/count.tsx (hunk 1 above)
Comment (rocketnova) [reply participant/app/routes/$localAgency/recertify/count.tsx:17]: @makaelastephens I've made some changes to make this and all the other pages more consistent and also to match the Figma more closely. Will you take another look?

File: participant/styles/_uswds-theme.scss (hunk 2 above)
Comment (rocketnova) [reply participant/styles/_uswds-theme.scss:18]: I'm glad you are checking this! This is an aspect of using Chromatic/storybook. On the dev environment, it looks like this:

Screen Shot 2023-03-27 at 12 07 06 PM

**Comment by makaelastephens** [comment]
Screen Shot 2023-03-28 at 1 18 03 PM
The spacing between the DOB field and "Are they enrolled in these other programs"? is very tight.

Screen Shot 2023-03-28 at 1 18 39 PM
Can we overwrite the line length for "How many people..." so it's similar to measure-6?

**Comment by rocketnova** [comment]
@makaelastephens 

> Screen Shot 2023-03-28 at 1 18 03 PM
> The spacing between the DOB field and "Are they enrolled in these other programs"? is very tight.

🤔 I'll take a look at that.

> Screen Shot 2023-03-28 at 1 18 39 PM
> Can we overwrite the line length for "How many people..." so it's similar to measure-6?

This is the USWDS standard form label max-width. Do you want to change it across the board?


Screen Shot 2023-03-28 at 11 14 42 AM

**Comment by makaelastephens** [comment]
@rocketnova 
> This is the USWDS standard form label max-width. Do you want to change it across the board?

Yes, we want to change it across the board. Checking the galaxy brain notes, unfortunately we didn't write down what we wanted the override to be so let me know if you need more eyes later.

**Comment by rocketnova** [comment]
> @makaelastephens
> 
> > Screen Shot 2023-03-28 at 1 18 03 PM
> > The spacing between the DOB field and "Are they enrolled in these other programs"? is very tight.
> 
> 🤔 I'll take a look at that.

@makaelastephens Turns out this is a known USWDS bug when you use legends without labels in fieldsets 😞 I'll do some custom CSS'ing to fix. Also fun: this affects chrome but not firefox

**Comment by danielladevera-nava** [comment]
The changes noted in the comments above now look as expected to me 👍.

**Review by makaelastephens — APPROVED** [review]
LGTM!

**Pull request merged**